source/extensions/filters/http/admission_control/config.cc

```diff
@@ -21,6 +22,10 @@ Http::FilterFactoryCb AdmissionControlFilterFactory::createFilterFactoryFromProt
     const envoy::extensions::filters::http::admission_control::v3alpha::AdmissionControl& config,
     const std::string& stats_prefix, Server::Configuration::FactoryContext& context) {
 
+  if (config.has_sr_threshold() && config.sr_threshold().default_value().value() == 0) {
+    throw EnvoyException("Success Rate Threshold cannot be zero percent");
```

Comment:
Can you add a config unit test to provide coverage of this line as well?

Reply:
Yup can do! Thanks!

Reply:
Added on Friday :)!